test/shell-pipeline-tests/README.md

```diff
@@ -21,15 +21,17 @@ For local execution, these environment variables need to be set:
 * **CUSTOM_IMAGE_REGISTRY_SECRET_PATH** - Directory where credentials for the image repository (e.g. Quay) are stored. This repository is used to publish the image after it is tested. The pipeline uses [basic-auth](https://tekton.dev/docs/pipelines/auth/#configuring-basic-auth-authentication-for-docker) for authentication. The directory should contain the files:
   * username - containing the username of the account used to access the image registry
   * password - containing the password used to access the image registry
+* **CUSTOM_GIT_CREDENTIALS_SECRET_PATH** - Directory where the GitHub token is stored. The directory should have 1 file:
```

Comment:
Is there a technical restriction for using GitHub and a personal access token if we want to verify this locally?  I assume use of Gitea or gitlab for verification would also work assuming they are completely compatible with the GitOps pipeline codebase

Reply:
@LaVLaS Not sure what you mean by `verify this locally`. We need a repository, which is reachable by the pipeline running on OpenShift, ideally one which can be accessed online using a browser for possible debugging etc.  Afaik our GitLab is behind VPN, isn't it? So it might not be reachable. I was hoping to use a separate GitHub repository with some kind of service account in the opendatahub-io org, there must be some already set up I expect.

Reply:
Ok. The task name `github-open-pr` does imply that the workflow is github only but it does work with GitHub and Gitea (I have not tried GitLab) using Personal Access Tokens.  The original comment was in reference to use of `GitHub` but given the task name, I can understand why it was used

Reply:
The task name having github specified in it is because it expects the GitHub API, which Gitea also happens to implement. I'm not sure if Gitlab or Bitbucket or other Git forges have the same API. If they don't, then a future improvement to the GitOps Update pipeline might be to add support for them.

I guess this credential description, and the token description on the following line, don't need to say _GitHub_ explicitly.

Reply:
OK, I changed it to:
```
* **CUSTOM_GIT_CREDENTIALS_SECRET_PATH** - Directory where the GitHub-compatible forge token is stored. The directory should have 1 file:
  * token - containing the GitHub-compatible forge token, e.g. for GitHub specifically it will have the form of `github_pat_123...`
```
I didn't want to say just Git token as GitLab token won't work for opening the PR and the pipeline will fail.